packages/slice-machine/pages/onboarding.tsx

```diff
@@ -16,26 +16,21 @@ import { LocalStorageKeys } from "@lib/consts";
 import router from "next/router";
 
 import { BiChevronLeft } from "react-icons/bi";
+import { Video as CldVideo } from "cloudinary-react";
 
-import BuildSliceVideo from "../public/onboarding-videos/build-slice.mp4";
-import AddPageVideo from "../public/onboarding-videos/add-to-page.mp4";
-import PushToPrismicVideo from "../public/onboarding-videos/push-to-prismic.mp4";
-
-const Video = (props: React.VideoHTMLAttributes<HTMLVideoElement>) => {
-  return (
-    <video
-      controls
-      autoPlay
-      loop
-      {...props}
-      style={{
-        maxWidth: "100%",
-        height: "auto",
-        ...props.style,
-      }}
-    />
-  );
-};
+const Video = (props: VideoProps) => (
+  <CldVideo
+    cloudName="dmtf1daqp"
```

Comment:
what is that ?

Reply:
cloudinary's video player, we decided to keep the videos on cloudinary